scripts/Gossip.js

```diff
@@ -40,28 +40,42 @@ var Gossip = function(robot) {
     });
   }
 
-  var findGossip = function(text, msg) {
+  var findGossip = function(text, msg, limiter) {
     var connection = connectionHelper.getConnection();
     var query = 'SELECT * FROM gossip';
     var queryArgs = [];
+    var gossip = '\n';
```

Comment:
You can probably put the `\n` into a constant somewhere, maybe call it `LINE_BREAK` or `LINE_DELIMITER`.